lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1039,7 +1023,10 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
 
   uint64_t getSizeOfBasicType(DebugTypeInfo DbgTy) {
     uint64_t SizeOfByte = CI.getTargetInfo().getCharWidth();
-    uint64_t BitWidth = DbgTy.getSize().getValue() * SizeOfByte;
+    uint64_t BitWidth = 0;
+    assert(DbgTy.getSize() && "non-fixed basic type");
```

Comment:
I take it you couldn't reasonably refactor `getSizeOfBasicType` to take a `CompletedDebugTypeInfo` instead?

Reply:
No, just laziness. Will be part of the next version.